src/foundry/foundry.js/gameTime.d.ts

```diff
@@ -3,7 +3,7 @@
  * Uses a basic implementation of https://www.geeksforgeeks.org/cristians-algorithm/ for synchronization.
  */
 declare class GameTime {
-  constructor(socket: io.Socket);
+  constructor(socket: io.Socket | null | undefined);
```

Comment:
if this can be undefined, wouldn't it make more sense to declare the parameter optional?